dashboard/app/views/parent_mailer/parent_permission_reminder.html.haml

```diff
@@ -14,7 +14,11 @@
   Code.org is a US-based nonprofit that provides free computer science courses and activities.  We collect limited personal data solely for the purpose of providing the services, including:  Display Name (we recommend using only first name - used to provide welcome); Username (not actual name); Passwords; Hashed Email Address (not stored in retrievable format - can't be used to contact student); Age (not birthday); and Gender (optional) (used for aggregate measurements).
 
 %p
-  We do not sell data or exploit it for financial gain. We do not sell ads.  We do not store student email addresses in a retrievable format.  For more information on Code.org accounts for children under 13 years old, please see our <u>Children's Privacy Notice</u>, which is part of our full
+  Code.org assigns utmost importance to student data privacy and security. As a not-for-profit organization, they are transparent about the limited data they collect and how they use it.  They do not sell data or exploit it for financial gain. They do not sell ads. Code.org has signed the
+  %a{href:"https://studentprivacypledge.org/signatories/"}Student Privacy Pledge
+  and their privacy practices are highly rated by
+  %a{href:"https://privacy.commonsense.org/evaluation/code.org"}Common Sense Media.
```

Comment:
It would be nice to move these links to [config.yml.erb](https://github.com/code-dot-org/code-dot-org/blob/staging/config.yml.erb)